assets/stylesheets/bookingsync/_sweetalert.scss

```diff
@@ -1,3 +1,8 @@
+.swal2-buttonswrapper {
```

Comment:
What if we pass a `customClass` to the modal? Would that allow us to not use `!important`?